routes/api/articles.js

```diff
@@ -9,7 +14,7 @@ router.put('/articles/:slug', authenticate, checkNullInput, articleController.up
 router.delete('/articles/:slug', authenticate, articleController.deleteArticle);
 router.get('/articles', authenticate, articleController.getAllArticles);
 router.get('/articles/:slug', authenticate, articleController.getAnArticle);
+router.post('/articles/:articleId/rating', authenticate, checkIfArticleExist, getUserRating, articleController.postRating);
 router.post('/articles/:slug/react', authenticate, reactionValidation, articleController.reactToArticle);
```

Comment:
So, I would suggest that we use only one validator here. Maybe combine checkIfArticleExists and get getUserRating. If getUserRating is not validating the data that has been posted then I think that it should be in your controller instead.

 TLDR I would have one validator here and move the getUserRating function call to the 'postRating' controller.